pkgs/development/python-modules/pwinput/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  lib,
+}:
+buildPythonPackage rec {
+  pname = "pwinput";
+  version = "1.0.3";
+
```

Comment:
Could you add `pyproject = true` to avoid a legacy `setup.py` build?

> Note that the pyproject format falls back to using setuptools, so you can use pyproject = true even if the package only has a setup.py.

> When unset, the legacy setuptools hooks are used for backwards compatibility.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters